WinUIGallery/ControlPages/InfoBadgePage.xaml

```diff
@@ -114,7 +115,8 @@
 
             <local:ControlExample.Xaml>
                 <x:String xml:space="preserve">
-&lt;Button Padding="0" Width="200" Height="60" HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch"&gt;
+&lt;Button Padding="0" Width="200" Height="60" ToolTipService.ToolTip="Refresh required"
+        HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch"&gt;
```

Comment:
HorizontalAlignment="Center" that is present above is missing from the text.

Reply:
Good catch, fixed.